README.md

```diff
@@ -50,7 +50,12 @@ If you click "cancel" before the import is complete, all pieces imported so far
 
 You can also update existing pieces via this module.
 
-To do that, you will need one (and only one) **key column** in your file. This column's name **must be exactly the name of the existing field** that uniquely identifies each row as an update of a specific existing piece, **followed by `:key`**.
+To do that, you will need one (and only one) **key column** in your file. This column's name **must be exactly the name of the existing field followed by `:key`**. This will identify each row as an update of a specific existing piece.
+
+### Using the `:key` column
```

Comment:
This makes it sound more and more like the column is actually called `:key` rather than, for instance, `username:key`. Maybe revise with that potential confusion in mind.